app/src/main/java/com/dongyang/android/youdongknowme/ui/view/depart/DepartActivity.kt

```diff
@@ -58,13 +36,21 @@ class DepartActivity : AppCompatActivity(), DepartClickListener {
             this.setHasFixedSize(true)
         }
 
-        // 선택 포지션을 실시간 옵저빙
+    }
+
+    override fun initDataBinding() {
+
+    }
```

Comment:
사용하지 않는 함수는 
```suggestion
    override fun initDataBinding() = Unit
```
으로 정리해주시면 좋을 것 같아요 << 정호씨가 알려줌